README.md

```diff
@@ -0,0 +1,48 @@
+## Notablog
+
+- A Fullstack ROR project
+
+## Build
+
+- built with Ruby
+
+### Setup
+
+- Clone the project
+
+```terminal
+git clone git@github.com:nottherealalanturing/notablog.git
+```
+
+- cd into the project directory
+
+```terminal
+cd notablog
+```
```

Comment:
- [ ] Hello and nice job, but please make sure that the readme file is having all details of the project :( what's the project, the installation, and the creation and migration of the data)